torch/csrc/utils/python_strings.h

```diff
@@ -98,6 +99,14 @@ inline void THPUtils_internStringInPlace(PyObject** obj) {
  */
 
 inline py::object PyObject_FastGetAttrString(PyObject* obj, const char* name) {
+#if IS_PYTHON_3_13_PLUS
+  PyObject* res = (PyObject*)nullptr;
```

Comment:
Huh, so the cast from nulptr necessary? Probably a holdover from C to C++ copy paste but probably better to save for a file wide cleanup